lib/scala/bench-processor/src/main/java/org/enso/benchmarks/processor/BenchProcessor.java

```diff
@@ -298,6 +298,8 @@ private static String normalize(String name) {
     for (char c : name.toCharArray()) {
       if (isValidChar(c)) {
         normalizedNameSb.append(c);
+      } else if (c == '-') {
+        normalizedNameSb.append("minus");
```

Comment:
Why this replacement? Won't `_` work?

Imagine a name `enso-test` being replaced as `ensominustest` - the result is not very readable.

If anything, let's do `_minus_`,  but IMO just `_` could be fine.

Reply:
On #7519, it is already ensured that all the benchmark names are valid Java identifiers - https://github.com/enso-org/enso/pull/7519/commits/6b670e9c1864c4a0f5d00c9d07b0bb4216675ba5. This renders this comment outdated.